backend/src/main/java/org/youcancook/gobong/domain/user/dto/request/SignupRequest.java

```diff
@@ -4,13 +4,14 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import org.youcancook.gobong.global.validation.annotation.UserNickname;
 
 @Getter
 @NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)
 @AllArgsConstructor
 public class SignupRequest {
 
-    @NotBlank(message = "nickname은 필수입니다.")
+    @UserNickname
```

Comment:
```suggestion
    @Pattern(regexp = "[0-9a-zA-Z가-힣]{1,10}")
```

Reply:
https://github.com/you-can-cook/Gobong/pull/22#discussion_r1289613898